templates/advantage/index.html

```diff
@@ -44,6 +44,15 @@ <h2>Your {% if subscriptions['total_subscriptions'] > 1 %}{{subscriptions['total
         </div>
       </div>
       {% if subscriptions['has_monthly'] %}
+        {% if not default_payment_method["id"] %}
+          <div class="row" style="margin-top: 1rem;">
```

Comment:
It's not good practise to put margins or padding on rows as they should just act as a clearfix. Maybe this needs to be within its own strip?

Reply:
@ClementChaumel correct me if I'm wrong, but I think this was the result of a long discussion and much back-and-forth with @anasereijo.

I think maybe @anthonydillon you should talk to @anasereijo directly, face to face, about this issue. Unless I've got the wrong end of the stick @ClementChaumel?

Reply:
I think you might be thinking of something else? :thinking:  I just added that here to match the design. I don't remember discussing this issue

Reply:
Ah then yes I'm thinking of something else =) ignore me.